Update rest_api.md (#4720)
apache/openwhisk: docs/rest_api.md

### Triggers with Feed Actions

> There are special triggers that can be created using a feed action. The feed action is an action that helps with the configuration of a feed provider that will be in charge of firing the trigger whenever there is an event for the trigger. Learn more about these feed providers in the [feeds.md] documentation.

Reviewer: ```suggestion
There are special triggers that can be created using a feed action. The feed action configures a feed provider such that events from the provider results in triggers being fired. Learn more about these feed providers in the [feeds.md] documentation.
```

Reviewer: While at it, WDYT of the alternate wording?
